datamodel.itop-standard-email-synchro.xml

```diff
@@ -833,7 +838,7 @@ EOF
 		}
 					
 		// Determine which field to update
-		$sAttCode = 'public_log';
+		$sAttCode = $this->Get('ticket_updates_log_attcode');
 		$aAttCodes = MetaModel::GetModuleSetting('itop-standard-email-synchro', 'ticket_log', array('UserRequest' => 'public_log', 'Incident' => 'public_log'));
```

Comment:
```suggestion
		$sAttCode = $this->Get('ticket_updates_log_attcode');
		$aAttCodes = MetaModel::GetModuleSetting('itop-standard-email-synchro', 'ticket_log', []);
```

Reply:
What about existing/upgrading users? They will still have the setting from previous install. Also new users will have this setting as it is still present in module file.